testsuite/smoke-test/src/aptos/staking.rs

```diff
@@ -1,97 +0,0 @@
-// Copyright (c) Aptos
-// SPDX-License-Identifier: Apache-2.0
-
-use crate::smoke_test_environment::new_local_swarm_with_aptos;
-use aptos_transaction_builder::aptos_stdlib;
-use aptos_types::{account_address::AccountAddress, account_config::CORE_CODE_ADDRESS};
-use forge::{AptosPublicInfo, Swarm};
-
-// re-enable after delegation is enabled
-#[ignore]
```

Comment:
This test has been ignored since the beginning. We now have a different smoke-test and e2e tests in e2e-move-tests to cover staking.